plugins/certs/subcommands/print

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eo pipefail
+[[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/certs/functions"
+
+cmd-certs-download() {
+  declare desc="exports SSL cert or key via STDOUT"
+  declare cmd="$1"
+  [[ "$1" == "$cmd" ]] && shift 1
+  declare APP="$1" KEY_TYPE="$2"
+
+  [[ -z "$APP" ]] && dokku_log_fail "Please specify an app to run the command on"
+  verify_app_name "$APP"
+  local APP_SSL_PATH="$DOKKU_ROOT/$APP/tls"
+
+  if [[ "$KEY_TYPE" != "key" ]]  && [[ "$KEY_TYPE" != "crt" ]]; then
+    dokku_log_fail "specify either 'key' or 'crt'"
+  fi
+
+  if ! is_ssl_enabled "$APP"; then
+    dokku_log_fail "$APP doesn't have an SSL endpoint defined"
+  fi
+
+  cat "$APP_SSL_PATH/server.$KEY_TYPE"
+}
+
+cmd-certs-download "$@"
```

Comment:
Can you add a newline at the end of the file?

Reply:
Also, should rename the function to `cmd-certs-print` if you named the command that. I still think `show` is a better command, and is more inline with what we've been naming things elsewhere (config:show, nginx:show-config...).

Reply:
It's awesome how quick you are with feedback. I've changed it to show.

Reply:
I just happen to be awake :P